### ISSUE TITLE
Crash if sending undefined

### ISSUE THREAD

**Issue by lordgreg** [hypery2k/cordova-email-plugin#27]
Using the plugin like that:
```
      var _EMAIL = ['valid@mail1.com', 'valid@mail2.com', 'valid@mail3.com', undefined, undefined];

      cordova.plugins.email.isAvailable(
        function (isAvailable) {
          if (!isAvailable) {
            return;
          }

          // service available...
          cordova.plugins.email.open({
            to: _EMAIL,
            subject: 'some title,
            body: 'some message'
          });
        }
      );
```

will crash the whole app.

**Comment by hypery2k** [comment]
can you tell more about you're setup, see [CONTRIBUTING.MD](https://github.com/hypery2k/cordova-email-plugin/blob/master/CONTRIBUTING.MD)

**Comment by lordgreg** [comment]
Sorry @hypery2k, of course:

I've tested this on iOS devices with iO9+

Cordova:

```
Cordova 6.4.0
<engine name="ios" spec="~4.2.1" />
```

Xcode 7.X
node v6.3.1

**Comment by hypery2k** [comment]
which version of the plugin you're using?

**Comment by lordgreg** [comment]
We are using the 1.2.0

**Comment by hypery2k** [comment]
can you try 1.2.1, see https://github.com/hypery2k/cordova-email-plugin/releases/tag/v1.2.1

**Comment by hypery2k** [comment]
could you try with `cordova plugin add cordova-plugin-email@next`?

**Comment by hypery2k** [comment]
fixed in 1.2.3
